pages/linux/logrotate.md

```diff
@@ -0,0 +1,24 @@
+# logrotate
+
+> Rotates, compresses, and mails system logs.
+> More information: <https://manned.org/logrotate.8>.
+
+- Force logrotate to run outside of normal interval:
+
+`logrotate {{path/to/logrotate.conf}} --force`
+
+- Set mail binary for logrotate to Mutt:
```

Comment:
Is Mutt something usually used with logrotate? If it's just a random example, we should probably keep it more general with `/usr/bin/mail_binary`.
```suggestion
- Set the mail binary for logrotate to Mutt:
```